src/app/components/experiment-workspace/experiment-workspace.component.html

```diff
@@ -9,41 +9,49 @@ <h1 class="t-page-headline">Experiment Workspace</h1>
                     <mat-icon>arrow_back</mat-icon>
                 </a -->
             </div>
-            <div *ngIf="activePlugin == null">
-                <p>Choose a plugin.</p>
+            <div *ngIf="!infoAreaActive">
```

Comment:
you can use `ng-container` instead of div when you want to avoid adding extra nodes to the dom just for an `*ngIf` or `*ngFor`